Articles/ShortArticles/TheDeveloperCertificateOfOrigin.md

```diff
@@ -0,0 +1,153 @@
+# The Developer Certificate of Origin
+
+<!--deck text start-->
+Every project that accepts contributions from outside developers should have a legal mechanism in place that allows those contributions to become part of the main code base that allows for further development and distribution and avoids future legal problems.
+The Developer Certificate of Origin with signoffs embedded into Git commits messages is becoming the preferred way to handle this issue and is being adopted by important open source projects.
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe)
+
+#### Publication date: ???, 2021
+
+When a person writes a piece of software, they own the copyrights in that software by default, unless it is a "work for hire", in which case their employer owns it, or those rights are conveyed to some other legal entity by a written agreement.
+Even if that person provides that source code with the implied intent to make it part of some existing software project or otherwise give it away (i.e. place it in the public domain), legal experts may differ as to whether that by itself is enough to allow the receiving project to modify and distribute the contribution along with the rest of the project.
+For some experts, contributions are implicitly offered under the same licensing terms as the code base to which the contribution is being made.
+Others prefer to have the owner of the copyright in new contributions explicitly state that they are providing the software (or a modification to an existing piece of software) to a project in accordance with some specific software license<sup>[0]</sup> associated with the project.
+It is the process by which a person provides an (updated) contribution to an existing project that has an existing software license that is the focus of this article.
+
+Until fairly recently, Contributor License Agreements (CLAs)<sup>[1],[2]</sup> have been the industry standard way to accept such contributions.
+CLAs can be signed by individual contributors or by organizations (to cover all of its members).
+While CLAs have a number of advantages for a project from a legal standpoint (such as allowing for later re-licensing), they are unpopular with some developers and developer communities and are seen by many as an impediment to accepting contributions<sup>[3],[5]</sup>.
```

Comment:
I suggest ^allowing^facilitating. You can relicense either way.  But if your CLA includes a relicensing clause, it will probably be a *lot* easier.

Reply:
Thanks!  Addressed in new commit ad48d7ea362084201f08e8da371dde0595dd11bf.